vaccine_feed_ingest/stages/ingest.py

```diff
@@ -324,17 +324,49 @@ def _validate_normalized(output_dir: pathlib.Path) -> bool:
                     return False
 
                 if normalized_location.location:
-                    if not BOUNDING_BOX.latitude.contains(
-                        normalized_location.location.latitude
-                    ) or not BOUNDING_BOX.longitude.contains(
-                        normalized_location.location.longitude
-                    ):
+                    result, failed_bound = validate_bounding_boxes(
+                        normalized_location.location, [BOUNDING_BOX, BOUNDING_BOX_GUAM]
```

Comment:
Can you either change the method to `validate_approved_bounds` and not have the caller pass in a list of bounds, or add a constant called `APPROVED_BOUNDS` which is a list of bounding boxes.

I want to make it easy for the the caller to do the right thing, and right now this flexibility means future callers might not pass all of the bounds.